milli/src/search/criteria/geo.rs

```diff
@@ -48,7 +48,14 @@ impl Criterion for Geo<'_> {
         // if there is no rtree we have nothing to returns
         let rtree = match self.rtree.as_ref() {
             Some(rtree) => rtree,
-            None => return Ok(None),
+            None => {
+                return Ok(Some(CriterionResult {
+                    query_tree: None,
+                    candidates: None,
+                    filtered_candidates: Some(params.excluded_candidates.clone()),
+                    bucket_candidates: Some(self.bucket_candidates.clone()),
+                }))
+            }
```

Comment:
Is it the solution chosen with @ManyTheFish?

Reply:
No sorry I was doing something else after our discussion with @ManyTheFish.
Now the chosen solution is implemented :smile: